### ISSUE TITLE
request.get_args method fails when querying child app

### ISSUE THREAD

**Issue by amaurymercier** [ashleysommer/sanic-dispatcher#11]
Hey (I know, it's me again !),

it seems the `sanic.request.Request` objects in the child app's routes are broken : they fail when calling `get_args`.

Package versions :
```
Sanic==19.3.1
Sanic-Dispatcher==0.7.1.0
```

Easy code to reproduce : `main.py`
```python
from sanic import Sanic
from sanic.response import text
from sanic_dispatcher import SanicDispatcherMiddlewareController

main_app = Sanic(__name__)
dispatcher = SanicDispatcherMiddlewareController(main_app)


@main_app.get('/')
def hello_main(request):
    """Route main."""
    return text('hello, main world !')


child_app = Sanic(__name__)


@child_app.route("/", methods=['GET'])
def hello_world(request):
    """Cors route."""
    return text("args : {}".format(str(request.args)))


dispatcher.register_sanic_application(child_app, '/child')

if __name__ == '__main__':
    main_app.run(port=5000, debug=True)
``` 

when calling 'http://localhost:5000/child/', the server raises a 500 :
```
[2019-06-24 14:16:27 +0200] [31179] [ERROR] Exception occurred while handling uri: 'http://localhost:5000/'
Traceback (most recent call last):
  File "/home/.../.virtualenvs/.../lib/python3.6/site-packages/sanic/app.py", line 915, in handle_request
    response = handler(request, *args, **kwargs)
  File "main.py", line 21, in hello_world
    return text("args : {}".format(str(request.args)))
  File "/home/.../.virtualenvs/.../lib/python3.6/site-packages/sanic/request.py", line 235, in get_args
    (keep_blank_values, strict_parsing, encoding, errors)
TypeError: 'NoneType' object is not subscriptable
```

Seems that `Request.parsed_args` is not properly set and is `None`..

Thanks again !

**Comment by ashleysommer** [comment]
~~Hi,
Yes, this is something I know is a problem of child apps, and is a feature I've been meaning to implement. Thanks for reminding me.~~
Thats not right, see next comment.

**Comment by ashleysommer** [comment]
Ok, turns out this feature _was_ implemented, but was broken in a change in Sanic 19.3 and above.
This was the commit. https://github.com/huge-success/sanic/commit/2a15583b87b441ffa7b386c5104705a8c1f65668
I know how to fix it, but it will require checking for which version of Sanic is running, and doing parsed_args differently.

**Comment by ashleysommer** [comment]
@amaurymercier Ok, please update to Sanic-Dispatcher v0.7.2.0 with this fix for Sanic >= 19.3.0

**Comment by amaurymercier** [comment]
I tested it, and it works! Thanks a lot!
